components/lib/nearby/build.gradle

```diff
@@ -0,0 +1,53 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+plugins {
+    id "com.jetbrains.python.envs" version "0.0.26"
+}
```

Comment:
nit: This plugin is not needed. It's required when a component needs to send telemetry via "glean".

Reply:
I appreciate the information. I'm not knowledgeable about gradle and dependencies (as should be evident).